src/main/resources/schema.sql

```diff
@@ -39,13 +39,13 @@ CREATE TABLE tourOffer
 );
 CREATE TABLE reservation
 (
-  id              NUMERIC IDENTITY PRIMARY KEY,
-  client_id       INT          NOT NULL,
-  tourOffer_id    INT          NOT NULL,
-  numberOfPersons INT          NOT NULL,
-  status          VARCHAR(100) NOT NULL,
-  discount_id     INT,
-  totalPrice      INT,
+  id             NUMERIC IDENTITY PRIMARY KEY,
+  client_id      INT          NOT NULL,
+  tourOffer_id   INT          NOT NULL,
+  numberOfPeople INT          NOT NULL,
```

Comment:
In DB we have some column naming conventions too. Please try to follow them, there is no camel case here

Reply:
done